submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+                  <input type="radio" name="tone" id="friendly" class="input" />
+                  <div class="select-tone tone-before chat">
+                    <span class="checkedTone"></span>Friendly / Casual
+                  </div>
+                </label>
+                <label for="corporate">
+                  <input
+                    type="radio"
+                    name="tone"
+                    id="corporate"
+                    class="input"
+                    checked
+                  />
+                  <div class="select-tone tone-before suitcase">
+                    <span class="checkedTone"></span>Corporate / Pro
+                  </div>
+                </label>
+                <label for="media">
+                  <input type="radio" name="tone" id="media" class="input" />
+                  <div class="select-tone tone-before media">
+                    <span class="checkedTone"></span>Media / Formal
+                  </div>
+                </label>
+              </div>
+              <div class="time-range">
+                <input
+                  type="range"
+                  class="time-range-button"
+                  min="0"
+                  max="100"
+                  step="1"
+                  value="50"
+                />
+                <div class="range-text">
+                  <h5 class="range-under">I GOT TIME</h5>
+                  <h5 class="range-under">AVERAGE</h5>
+                  <h5 class="range-under">YESTERDAY</h5>
+                </div>
+                <div class="range-info">
+                  <h4 class="info-header">TIMELINE INFO</h4>
+                  <p class="range-para">
+                    With an <span class="to-orange">AVERAGE</span> timeline set
+                    up, our expert translator can take a reasonable amount of
+                    time perfecting your translation
+                  </p>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="payment-method">
+            <div class="allTextDocumentUpload">
+              <span class="numberSection"><b>5</b></span>
+              <h2 class="uploadText">Select your payment method</h2>
+            </div>
+            <div class="block-to-center">
+              <div>
+                <h5 class="smallUploadText">
+                  Already have an account with us?
+                  <a href="#" class="underlined-a">Sign in.</a>
+                </h5>
+                <h5 class="above-text">FULL NAME</h5>
+                <input
+                  type="text"
+                  id="text-name"
+                  class="text text-name"
+                  name="Full Name"
+                  placeholder="Mattew Alebard"
+                  title="Enter your name. Minimal length 3 characters."
+                  minlength="3"
+                  required
+                /><span class="validation-sign"></span>
```

Comment:
Try to use `::after` for `validation-sign`

Reply:
Actually, I am using validation-sign with ::after. If you are talking about using validation sign like pseudo-element for `<input>` tag, I've read that It is not working for `<input>`